docs/modules/qbi-helper/interest-template.md

```diff
@@ -0,0 +1,141 @@
+<%(TOC)%>
+
+# How to use InterestTemplate
+
+<%(Callout message="
+Before reading this document, make sure you have read:
+
+  * [Introduction to Query-based interest]({{urlRoot}}/modules/qbi-helper/intro-to-qbi)
+")%>
+
+An [`InterestTemplate`]({{urlRoot}}/api/query-based-interest/interest-template) is effectively a wrapper around the `Interest` component, providing ways to more easily add, replace and clear queries from the underlying interest map.
+
+## Initialise
```

Comment:
Its kinda weird the heading is called `Initialise` but really you are Creating a template.